js/code-tabs.js

```diff
@@ -0,0 +1,172 @@
+ * # Getting started with Spine in Kotlin
+ * </div>
+ * ```
+ *
+ * To change only some of the words in a sentence, use the `<span>` tag with the `.inline` class:
+ * ```
+ * A minimal client-server application in
+ * <span class="code-tab-content inline java">Java</span>
+ * <span class="code-tab-content inline kotlin">Kotlin</span>
+ * which handles one command to print some text...
+ * ```
+ */
+
+'use strict';
+
+$(
+    function() {
+        const cookieCodeLang = 'codeLang';
+        const $codeTabs = $('.code-tabs');
+        const $codeTabContent = $('.code-tab-content');
+
+        // TODO:2021-03-17:juliaevseeva: Update the logic to make it easy to add new programming languages.
+        const primaryLang = 'java';
+        const secondaryLang = 'kotlin';
+
+        addTabSwitcher();
+        initCodeLangSwitcher();
+
+        /**
+         * Adds a tab switcher to the DOM.
+         */
+        function addTabSwitcher() {
+            $codeTabs.each(function() {
+                const tabBlock = $(this);
+                createTab(tabBlock, createTabContainer(tabBlock));
+            });
+        }
+
+        /**
+         * Creates a container for tabs.
+         *
+         * @param tabBlock a block that contains tabs.
+         * @return {jQuery|HTMLElement} tabContainer a container for tabs.
+         */
+        function createTabContainer(tabBlock) {
+            const tabContainer = $('<div class="tabs"></div>');
+            tabBlock.prepend(tabContainer);
+            return tabContainer;
+        }
+
+        /**
+         * Creates a tab inside the container for each `code-tab-content` element.
+         *
+         * @param tabBlock a block that contains tabs.
+         * @param tabContainer a container for tabs.
+         */
+        function createTab(tabBlock, tabContainer) {
+            const tabContent = tabBlock.find($codeTabContent);
+            tabContent.each(function () {
+                const tabName = $(this).attr('class').split(' ')[1];
+                const item = $(`<div class="tab ${tabName}">${tabName}</div>`);
+                tabContainer.append(item);
+            });
+        }
+
+        /**
+         * Inits a code language switcher.
+         *
+         * By default, sets the primary code language to the `cookie`.
+         * On a tab click switches between code languages.
+         */
+        function initCodeLangSwitcher() {
+            const cookieValue = Cookies.get(cookieCodeLang);
+
+            if (cookieValue == null) {
+                setCodeLang(primaryLang);
+            } else {
+                primaryLang === cookieValue && setCodeLang(primaryLang);
+                secondaryLang === cookieValue && setCodeLang(secondaryLang);
+            }
+
+            $('.tab').click(function() {
+                const lang = $(this).attr('class').split(' ')[1];
+
+                if (lang === primaryLang) {
+                    setCodeLang(primaryLang);
+                } else {
```

Comment:
This and other `if/else` blocks above make adding yet another language impossible. Please code for having at least 4 languages. 

Please also encapsulate the knowledge of this (the names of the languages, and a number of them) in one place.